File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        std::unordered_map<int, int> count;

        int target = nums.size() / 2;
        for(const auto& num : nums) {
            count[num]++;
            if(count[num] > target) {
                return num;
            }
        }
        return 0;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        num_set = {}
        
        for ind, n in enumerate(nums):
            for k in num_set.keys():
                if n+k == target:
                    return [num_set[k], ind]

            if n not in num_set.keys():
                num_set[n] = ind
            